docs/labs/1-JavaEclipse.md

```diff
@@ -26,6 +27,55 @@ You can do this by:
 1. Visiting your add and remove programs setting in windows (or applications on a mac)
 2. Searching for any java programs
 3. **Removing** any existing java installations.
+4. Or you can check out these instruction.
```

Comment:
please add a heading before your section called:

### How to remove Java from your machine